packages/pd-manager-rest-api/src/pd-manager-api-server.ts

```diff
@@ -0,0 +1,59 @@
+import { agentContext } from '@sphereon/ssi-sdk.core'
+import { TAgent } from '@veramo/core'
+
+import express, { Express, Router } from 'express'
+import { IPDManagerAPIEndpointOpts, IRequiredPlugins } from './types'
+import { pdAddEndpoint, pdDeleteEndpoint, pdPersistEndpoint, pdReadEndpoint, pdUpdateEndpoint } from './api-functions'
+import { copyGlobalAuthToEndpoints, ExpressSupport } from '@sphereon/ssi-express-support'
+
+export class PdManagerApiServer {
+  private readonly _express: Express
+  private readonly _agent: TAgent<IRequiredPlugins>
+  private readonly _opts?: IPDManagerAPIEndpointOpts
+  private readonly _router: Router
+
+  constructor(args: { agent: TAgent<IRequiredPlugins>; expressSupport: ExpressSupport; opts?: IPDManagerAPIEndpointOpts }) {
```

Comment:
create a type for the args? a bit nicer?

Reply:
Fixed, but not in all other ApiServers